From 9d341d97c1eaad0e5b6dc6c0f49c59e7d2134151 Mon Sep 17 00:00:00 2001
From: Juan Nunez-Iglesias <jni@fastmail.com>
Date: Mon, 8 May 2023 20:54:48 +1000
Subject: [PATCH] test: [Automatic] Constraints upgrades: `certifi`, `dask`,
 `fsspec`, `hypothesis`, `imageio`, `ipython`, `pint`, `qtconsole`, `rich`,
 `virtualenv` (#5788)

This PR is automatically created and updated by this napari GitHub
Action:

https://github.com/napari/napari/tree/main/.github/workflows/upgrade_test_contraints.yml
It ensures that dependencies are tested with the most recent version.

The updated packages are:

 * certifi
 * coverage
 * dask
 * freetype-py
 * fsspec
 * hypothesis
 * imageio
 * ipython
 * lit
 * pint
 * platformdirs
 * qtconsole
 * requests
 * rich
 * typer
 * urllib3
 * virtualenv

Co-authored-by: alisterburt <alisterburt@users.noreply.github.com>
---
 resources/constraints/constraints_py3.10.txt | 88 ++++++++-----------
 resources/constraints/constraints_py3.8.txt  | 89 ++++++++-----------
 resources/constraints/constraints_py3.9.txt  | 92 +++++++++-----------
 3 files changed, 114 insertions(+), 155 deletions(-)

diff --git a/resources/constraints/constraints_py3.10.txt b/resources/constraints/constraints_py3.10.txt
index 2e0b30cc..40b894ca 100644
--- a/resources/constraints/constraints_py3.10.txt
+++ b/resources/constraints/constraints_py3.10.txt
@@ -42,11 +42,11 @@ click==8.1.3
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.26.4
+cmake==3.26.3
     # via triton
 comm==0.1.3
     # via ipykernel
-contourpy==1.1.0
+contourpy==1.0.7
     # via matplotlib
 coverage==7.2.7
     # via pytest-cov
@@ -74,14 +74,14 @@ executing==1.2.0
     # via stack-data
 fasteners==0.18
     # via zarr
-filelock==3.12.2
+filelock==3.12.0
     # via
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.40.0
+fonttools==4.39.3
     # via matplotlib
-freetype-py==2.4.0
+freetype-py==2.3.0
     # via vispy
 fsspec==2023.6.0
     # via
@@ -102,13 +102,13 @@ imageio==2.31.1
     #   scikit-image
 imagesize==1.4.1
     # via sphinx
-importlib-metadata==6.7.0
+importlib-metadata==6.6.0
     # via dask
 in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.23.2
+ipykernel==6.22.0
     # via
     #   napari-console
     #   qtconsole
@@ -132,7 +132,7 @@ jupyter-client==8.2.0
     # via
     #   ipykernel
     #   qtconsole
-jupyter-core==5.3.1
+jupyter-core==5.3.0
     # via
     #   ipykernel
     #   jupyter-client
@@ -153,9 +153,9 @@ lxml==4.9.2
     # via napari (setup.cfg)
 magicgui==0.7.2
     # via napari (setup.cfg)
-markdown-it-py==3.0.0
+markdown-it-py==2.2.0
     # via rich
-markupsafe==2.1.3
+markupsafe==2.1.2
     # via jinja2
 matplotlib==3.7.1
     # via napari (setup.cfg)
@@ -176,9 +176,7 @@ napari-plugin-engine==0.2.0
     #   napari (setup.cfg)
     #   napari-console
     #   napari-svg
-napari-plugin-manager==0.1.0a2
-    # via napari (setup.cfg)
-napari-svg==0.1.10
+napari-svg==0.1.6
     # via napari (setup.cfg)
 nest-asyncio==1.5.6
     # via ipykernel
@@ -187,12 +185,10 @@ networkx==3.1
     #   scikit-image
     #   torch
 npe2==0.7.0
-    # via
-    #   napari (setup.cfg)
-    #   napari-plugin-manager
+    # via napari (setup.cfg)
 numcodecs==0.11.0
     # via zarr
-numpy==1.25.0
+numpy==1.24.3
     # via
     #   contourpy
     #   dask
@@ -252,7 +248,7 @@ packaging==23.1
     #   superqt
     #   vispy
     #   xarray
-pandas==2.0.2 ; python_version >= "3.9"
+pandas==2.0.1 ; python_version >= "3.9"
     # via
     #   napari (setup.cfg)
     #   xarray
@@ -277,7 +273,7 @@ platformdirs==3.7.0
     #   jupyter-core
     #   pooch
     #   virtualenv
-pluggy==1.2.0
+pluggy==1.0.0
     # via pytest
 pooch==1.7.0
     # via
@@ -289,7 +285,7 @@ psutil==5.9.5
     # via
     #   ipykernel
     #   napari (setup.cfg)
-psygnal==0.9.1
+psygnal==0.9.0
     # via
     #   app-model
     #   magicgui
@@ -299,7 +295,7 @@ ptyprocess==0.7.0
     # via pexpect
 pure-eval==0.2.2
     # via stack-data
-pydantic==1.10.9
+pydantic==1.10.7
     # via
     #   app-model
     #   napari (setup.cfg)
@@ -313,10 +309,8 @@ pygments==2.15.1
     #   sphinx
     #   superqt
 pyopengl==3.1.6
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
-pyparsing==3.1.0
+    # via napari (setup.cfg)
+pyparsing==3.0.9
     # via matplotlib
 pyproject-hooks==1.0.0
     # via build
@@ -326,9 +320,9 @@ pyqt5-qt5==5.15.2
     # via pyqt5
 pyqt5-sip==12.12.1
     # via pyqt5
-pyqt6==6.5.1
+pyqt6==6.5.0
     # via napari (setup.cfg)
-pyqt6-qt6==6.5.1
+pyqt6-qt6==6.5.0
     # via pyqt6
 pyqt6-sip==13.5.1
     # via pyqt6
@@ -346,7 +340,7 @@ pyside6-essentials==6.4.2
     # via
     #   pyside6
     #   pyside6-addons
-pytest==7.3.2
+pytest==7.3.1
     # via
     #   napari (setup.cfg)
     #   pytest-cov
@@ -354,13 +348,13 @@ pytest==7.3.2
     #   pytest-metadata
     #   pytest-pretty
     #   pytest-qt
-pytest-cov==4.1.0
+pytest-cov==4.0.0
     # via
     #   -r resources/constraints/version_denylist.txt
     #   napari (setup.cfg)
 pytest-json-report==1.5.0
     # via -r resources/constraints/version_denylist.txt
-pytest-metadata==3.0.0
+pytest-metadata==2.0.4
     # via pytest-json-report
 pytest-pretty==1.2.0
     # via napari (setup.cfg)
@@ -382,12 +376,12 @@ pyyaml==6.0
     #   dask
     #   napari (setup.cfg)
     #   npe2
-pyzmq==25.1.0
+pyzmq==25.0.2
     # via
     #   ipykernel
     #   jupyter-client
     #   qtconsole
-qtconsole==5.4.3
+qtconsole==5.4.1
     # via
     #   napari (setup.cfg)
     #   napari-console
@@ -396,7 +390,6 @@ qtpy==2.3.1
     #   magicgui
     #   napari (setup.cfg)
     #   napari-console
-    #   napari-plugin-manager
     #   qtconsole
     #   superqt
 requests==2.31.0
@@ -408,7 +401,7 @@ rich==13.4.2
     #   napari (setup.cfg)
     #   npe2
     #   pytest-pretty
-scikit-image==0.21.0
+scikit-image==0.20.0
     # via napari (setup.cfg)
 scipy==1.10.1 ; python_version >= "3.9"
     # via
@@ -451,13 +444,10 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-    #   napari-plugin-manager
-sympy==1.12
+sympy==1.11.1
     # via torch
-tensorstore==0.1.37
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
+tensorstore==0.1.36
+    # via napari (setup.cfg)
 tifffile==2023.4.12
     # via
     #   napari (setup.cfg)
@@ -473,11 +463,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.1
+torch==2.0.0
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.2
+tornado==6.3.1
     # via
     #   ipykernel
     #   jupyter-client
@@ -494,19 +484,17 @@ traitlets==5.9.0
     #   qtconsole
 triton==2.0.0
     # via torch
-typer==0.9.0
+typer==0.7.0
     # via npe2
-typing-extensions==4.6.3
+typing-extensions==4.5.0
     # via
     #   app-model
     #   magicgui
     #   napari (setup.cfg)
-    #   pint
     #   psygnal
     #   pydantic
     #   superqt
     #   torch
-    #   typer
 tzdata==2023.3
     # via pandas
 urllib3==2.0.3
@@ -529,17 +517,15 @@ wheel==0.40.0
     #   nvidia-nvtx-cu11
 wrapt==1.15.0
     # via napari (setup.cfg)
-xarray==2023.5.0
+xarray==2023.4.2
     # via napari (setup.cfg)
-zarr==2.15.0
+zarr==2.14.2
     # via napari (setup.cfg)
 zipp==3.15.0
     # via importlib-metadata

 # The following packages are considered to be unsafe in a requirements file:
-pip==23.1.2
-    # via napari-plugin-manager
-setuptools==68.0.0
+setuptools==67.7.2
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.8.txt b/resources/constraints/constraints_py3.8.txt
index 32c7ea12..7d3acd38 100644
--- a/resources/constraints/constraints_py3.8.txt
+++ b/resources/constraints/constraints_py3.8.txt
@@ -30,7 +30,7 @@ build==0.10.0
     # via npe2
 cachey==0.2.1
     # via napari (setup.cfg)
-certifi==2023.5.7
+certifi==2022.12.7
     # via
     #   napari (setup.cfg)
     #   requests
@@ -42,11 +42,11 @@ click==8.1.3
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.26.4
+cmake==3.26.3
     # via triton
 comm==0.1.3
     # via ipykernel
-contourpy==1.1.0
+contourpy==1.0.7
     # via matplotlib
 coverage==7.2.7
     # via pytest-cov
@@ -74,14 +74,14 @@ executing==1.2.0
     # via stack-data
 fasteners==0.18
     # via zarr
-filelock==3.12.2
+filelock==3.12.0
     # via
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.40.0
+fonttools==4.39.3
     # via matplotlib
-freetype-py==2.4.0
+freetype-py==2.3.0
     # via vispy
 fsspec==2023.6.0
     # via
@@ -102,7 +102,7 @@ imageio==2.31.1
     #   scikit-image
 imagesize==1.4.1
     # via sphinx
-importlib-metadata==6.7.0
+importlib-metadata==6.6.0
     # via
     #   dask
     #   jupyter-client
@@ -115,11 +115,11 @@ in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.23.2
+ipykernel==6.22.0
     # via
     #   napari-console
     #   qtconsole
-ipython==8.12.2
+ipython==8.12.0
     # via
     #   ipykernel
     #   napari (setup.cfg)
@@ -139,7 +139,7 @@ jupyter-client==8.2.0
     # via
     #   ipykernel
     #   qtconsole
-jupyter-core==5.3.1
+jupyter-core==5.3.0
     # via
     #   ipykernel
     #   jupyter-client
@@ -160,9 +160,9 @@ lxml==4.9.2
     # via napari (setup.cfg)
 magicgui==0.7.2
     # via napari (setup.cfg)
-markdown-it-py==3.0.0
+markdown-it-py==2.2.0
     # via rich
-markupsafe==2.1.3
+markupsafe==2.1.2
     # via jinja2
 matplotlib==3.7.1
     # via napari (setup.cfg)
@@ -183,9 +183,7 @@ napari-plugin-engine==0.2.0
     #   napari (setup.cfg)
     #   napari-console
     #   napari-svg
-napari-plugin-manager==0.1.0a2
-    # via napari (setup.cfg)
-napari-svg==0.1.10
+napari-svg==0.1.6
     # via napari (setup.cfg)
 nest-asyncio==1.5.6
     # via ipykernel
@@ -194,9 +192,7 @@ networkx==3.1
     #   scikit-image
     #   torch
 npe2==0.7.0
-    # via
-    #   napari (setup.cfg)
-    #   napari-plugin-manager
+    # via napari (setup.cfg)
 numcodecs==0.11.0
     # via zarr
 numpy==1.24.3
@@ -259,7 +255,7 @@ packaging==23.1
     #   superqt
     #   vispy
     #   xarray
-pandas==2.0.2 ; python_version < "3.9"
+pandas==2.0.1 ; python_version < "3.9"
     # via
     #   napari (setup.cfg)
     #   xarray
@@ -286,7 +282,7 @@ platformdirs==3.7.0
     #   jupyter-core
     #   pooch
     #   virtualenv
-pluggy==1.2.0
+pluggy==1.0.0
     # via pytest
 pooch==1.7.0
     # via
@@ -298,7 +294,7 @@ psutil==5.9.5
     # via
     #   ipykernel
     #   napari (setup.cfg)
-psygnal==0.9.1
+psygnal==0.9.0
     # via
     #   app-model
     #   magicgui
@@ -308,7 +304,7 @@ ptyprocess==0.7.0
     # via pexpect
 pure-eval==0.2.2
     # via stack-data
-pydantic==1.10.9
+pydantic==1.10.7
     # via
     #   app-model
     #   napari (setup.cfg)
@@ -322,10 +318,8 @@ pygments==2.15.1
     #   sphinx
     #   superqt
 pyopengl==3.1.6
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
-pyparsing==3.1.0
+    # via napari (setup.cfg)
+pyparsing==3.0.9
     # via matplotlib
 pyproject-hooks==1.0.0
     # via build
@@ -335,9 +329,9 @@ pyqt5-qt5==5.15.2
     # via pyqt5
 pyqt5-sip==12.12.1
     # via pyqt5
-pyqt6==6.5.1
+pyqt6==6.5.0
     # via napari (setup.cfg)
-pyqt6-qt6==6.5.1
+pyqt6-qt6==6.5.0
     # via pyqt6
 pyqt6-sip==13.5.1
     # via pyqt6
@@ -355,7 +349,7 @@ pyside6-essentials==6.3.1
     # via
     #   pyside6
     #   pyside6-addons
-pytest==7.3.2
+pytest==7.3.1
     # via
     #   napari (setup.cfg)
     #   pytest-cov
@@ -363,13 +357,13 @@ pytest==7.3.2
     #   pytest-metadata
     #   pytest-pretty
     #   pytest-qt
-pytest-cov==4.1.0
+pytest-cov==4.0.0
     # via
     #   -r resources/constraints/version_denylist.txt
     #   napari (setup.cfg)
 pytest-json-report==1.5.0
     # via -r resources/constraints/version_denylist.txt
-pytest-metadata==3.0.0
+pytest-metadata==2.0.4
     # via pytest-json-report
 pytest-pretty==1.2.0
     # via napari (setup.cfg)
@@ -393,12 +387,12 @@ pyyaml==6.0
     #   dask
     #   napari (setup.cfg)
     #   npe2
-pyzmq==25.1.0
+pyzmq==25.0.2
     # via
     #   ipykernel
     #   jupyter-client
     #   qtconsole
-qtconsole==5.4.3
+qtconsole==5.4.1
     # via
     #   napari (setup.cfg)
     #   napari-console
@@ -407,7 +401,6 @@ qtpy==2.3.1
     #   magicgui
     #   napari (setup.cfg)
     #   napari-console
-    #   napari-plugin-manager
     #   qtconsole
     #   superqt
 requests==2.31.0
@@ -419,9 +412,9 @@ rich==13.4.2
     #   napari (setup.cfg)
     #   npe2
     #   pytest-pretty
-scikit-image==0.21.0
+scikit-image==0.20.0
     # via napari (setup.cfg)
-scipy==1.10.1 ; python_version < "3.9"
+scipy==1.9.1 ; python_version < "3.9"
     # via
     #   napari (setup.cfg)
     #   scikit-image
@@ -462,13 +455,10 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-    #   napari-plugin-manager
-sympy==1.12
+sympy==1.11.1
     # via torch
-tensorstore==0.1.37
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
+tensorstore==0.1.36
+    # via napari (setup.cfg)
 tifffile==2023.4.12
     # via
     #   napari (setup.cfg)
@@ -484,11 +474,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.1
+torch==2.0.0
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.2
+tornado==6.3.1
     # via
     #   ipykernel
     #   jupyter-client
@@ -505,9 +495,9 @@ traitlets==5.9.0
     #   qtconsole
 triton==2.0.0
     # via torch
-typer==0.9.0
+typer==0.7.0
     # via npe2
-typing-extensions==4.6.3
+typing-extensions==4.5.0
     # via
     #   app-model
     #   ipython
@@ -518,7 +508,6 @@ typing-extensions==4.6.3
     #   rich
     #   superqt
     #   torch
-    #   typer
 tzdata==2023.3
     # via pandas
 urllib3==2.0.3
@@ -543,7 +532,7 @@ wrapt==1.15.0
     # via napari (setup.cfg)
 xarray==2023.1.0
     # via napari (setup.cfg)
-zarr==2.15.0
+zarr==2.14.2
     # via napari (setup.cfg)
 zipp==3.15.0
     # via
@@ -551,9 +540,7 @@ zipp==3.15.0
     #   importlib-resources

 # The following packages are considered to be unsafe in a requirements file:
-pip==23.1.2
-    # via napari-plugin-manager
-setuptools==68.0.0
+setuptools==67.7.2
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
diff --git a/resources/constraints/constraints_py3.9.txt b/resources/constraints/constraints_py3.9.txt
index a2521a3a..e3aac73f 100644
--- a/resources/constraints/constraints_py3.9.txt
+++ b/resources/constraints/constraints_py3.9.txt
@@ -30,7 +30,7 @@ build==0.10.0
     # via npe2
 cachey==0.2.1
     # via napari (setup.cfg)
-certifi==2023.5.7
+certifi==2022.12.7
     # via
     #   napari (setup.cfg)
     #   requests
@@ -42,11 +42,11 @@ click==8.1.3
     #   typer
 cloudpickle==2.2.1
     # via dask
-cmake==3.26.4
+cmake==3.26.3
     # via triton
 comm==0.1.3
     # via ipykernel
-contourpy==1.1.0
+contourpy==1.0.7
     # via matplotlib
 coverage==7.2.7
     # via pytest-cov
@@ -74,14 +74,14 @@ executing==1.2.0
     # via stack-data
 fasteners==0.18
     # via zarr
-filelock==3.12.2
+filelock==3.12.0
     # via
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.40.0
+fonttools==4.39.3
     # via matplotlib
-freetype-py==2.4.0
+freetype-py==2.3.0
     # via vispy
 fsspec==2023.6.0
     # via
@@ -102,7 +102,7 @@ imageio==2.31.1
     #   scikit-image
 imagesize==1.4.1
     # via sphinx
-importlib-metadata==6.7.0
+importlib-metadata==6.6.0
     # via
     #   dask
     #   jupyter-client
@@ -113,7 +113,7 @@ in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.23.2
+ipykernel==6.22.0
     # via
     #   napari-console
     #   qtconsole
@@ -137,7 +137,7 @@ jupyter-client==8.2.0
     # via
     #   ipykernel
     #   qtconsole
-jupyter-core==5.3.1
+jupyter-core==5.3.0
     # via
     #   ipykernel
     #   jupyter-client
@@ -158,9 +158,9 @@ lxml==4.9.2
     # via napari (setup.cfg)
 magicgui==0.7.2
     # via napari (setup.cfg)
-markdown-it-py==3.0.0
+markdown-it-py==2.2.0
     # via rich
-markupsafe==2.1.3
+markupsafe==2.1.2
     # via jinja2
 matplotlib==3.7.1
     # via napari (setup.cfg)
@@ -181,9 +181,7 @@ napari-plugin-engine==0.2.0
     #   napari (setup.cfg)
     #   napari-console
     #   napari-svg
-napari-plugin-manager==0.1.0a2
-    # via napari (setup.cfg)
-napari-svg==0.1.10
+napari-svg==0.1.6
     # via napari (setup.cfg)
 nest-asyncio==1.5.6
     # via ipykernel
@@ -192,12 +190,10 @@ networkx==3.1
     #   scikit-image
     #   torch
 npe2==0.7.0
-    # via
-    #   napari (setup.cfg)
-    #   napari-plugin-manager
+    # via napari (setup.cfg)
 numcodecs==0.11.0
     # via zarr
-numpy==1.25.0
+numpy==1.24.3
     # via
     #   contourpy
     #   dask
@@ -257,7 +253,7 @@ packaging==23.1
     #   superqt
     #   vispy
     #   xarray
-pandas==2.0.2 ; python_version >= "3.9"
+pandas==2.0.1 ; python_version >= "3.9"
     # via
     #   napari (setup.cfg)
     #   xarray
@@ -282,7 +278,7 @@ platformdirs==3.7.0
     #   jupyter-core
     #   pooch
     #   virtualenv
-pluggy==1.2.0
+pluggy==1.0.0
     # via pytest
 pooch==1.7.0
     # via
@@ -294,7 +290,7 @@ psutil==5.9.5
     # via
     #   ipykernel
     #   napari (setup.cfg)
-psygnal==0.9.1
+psygnal==0.9.0
     # via
     #   app-model
     #   magicgui
@@ -304,7 +300,7 @@ ptyprocess==0.7.0
     # via pexpect
 pure-eval==0.2.2
     # via stack-data
-pydantic==1.10.9
+pydantic==1.10.7
     # via
     #   app-model
     #   napari (setup.cfg)
@@ -318,10 +314,8 @@ pygments==2.15.1
     #   sphinx
     #   superqt
 pyopengl==3.1.6
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
-pyparsing==3.1.0
+    # via napari (setup.cfg)
+pyparsing==3.0.9
     # via matplotlib
 pyproject-hooks==1.0.0
     # via build
@@ -331,9 +325,9 @@ pyqt5-qt5==5.15.2
     # via pyqt5
 pyqt5-sip==12.12.1
     # via pyqt5
-pyqt6==6.5.1
+pyqt6==6.5.0
     # via napari (setup.cfg)
-pyqt6-qt6==6.5.1
+pyqt6-qt6==6.5.0
     # via pyqt6
 pyqt6-sip==13.5.1
     # via pyqt6
@@ -351,7 +345,7 @@ pyside6-essentials==6.3.1
     # via
     #   pyside6
     #   pyside6-addons
-pytest==7.3.2
+pytest==7.3.1
     # via
     #   napari (setup.cfg)
     #   pytest-cov
@@ -359,13 +353,13 @@ pytest==7.3.2
     #   pytest-metadata
     #   pytest-pretty
     #   pytest-qt
-pytest-cov==4.1.0
+pytest-cov==4.0.0
     # via
     #   -r resources/constraints/version_denylist.txt
     #   napari (setup.cfg)
 pytest-json-report==1.5.0
     # via -r resources/constraints/version_denylist.txt
-pytest-metadata==3.0.0
+pytest-metadata==2.0.4
     # via pytest-json-report
 pytest-pretty==1.2.0
     # via napari (setup.cfg)
@@ -387,12 +381,12 @@ pyyaml==6.0
     #   dask
     #   napari (setup.cfg)
     #   npe2
-pyzmq==25.1.0
+pyzmq==25.0.2
     # via
     #   ipykernel
     #   jupyter-client
     #   qtconsole
-qtconsole==5.4.3
+qtconsole==5.4.1
     # via
     #   napari (setup.cfg)
     #   napari-console
@@ -401,7 +395,6 @@ qtpy==2.3.1
     #   magicgui
     #   napari (setup.cfg)
     #   napari-console
-    #   napari-plugin-manager
     #   qtconsole
     #   superqt
 requests==2.31.0
@@ -413,9 +406,9 @@ rich==13.4.2
     #   napari (setup.cfg)
     #   npe2
     #   pytest-pretty
-scikit-image==0.21.0
+scikit-image==0.20.0
     # via napari (setup.cfg)
-scipy==1.10.1 ; python_version >= "3.9"
+scipy==1.9.1 ; python_version >= "3.9"
     # via
     #   napari (setup.cfg)
     #   scikit-image
@@ -456,13 +449,10 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-    #   napari-plugin-manager
-sympy==1.12
+sympy==1.11.1
     # via torch
-tensorstore==0.1.37
-    # via
-    #   -r resources/constraints/version_denylist.txt
-    #   napari (setup.cfg)
+tensorstore==0.1.36
+    # via napari (setup.cfg)
 tifffile==2023.4.12
     # via
     #   napari (setup.cfg)
@@ -478,11 +468,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.1
+torch==2.0.0
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.2
+tornado==6.3.1
     # via
     #   ipykernel
     #   jupyter-client
@@ -499,20 +489,18 @@ traitlets==5.9.0
     #   qtconsole
 triton==2.0.0
     # via torch
-typer==0.9.0
+typer==0.7.0
     # via npe2
-typing-extensions==4.6.3
+typing-extensions==4.5.0
     # via
     #   app-model
     #   ipython
     #   magicgui
     #   napari (setup.cfg)
-    #   pint
     #   psygnal
     #   pydantic
     #   superqt
     #   torch
-    #   typer
 tzdata==2023.3
     # via pandas
 urllib3==2.0.3
@@ -535,9 +523,9 @@ wheel==0.40.0
     #   nvidia-nvtx-cu11
 wrapt==1.15.0
     # via napari (setup.cfg)
-xarray==2023.5.0
+xarray==2023.4.2
     # via napari (setup.cfg)
-zarr==2.15.0
+zarr==2.14.2
     # via napari (setup.cfg)
 zipp==3.15.0
     # via
@@ -545,9 +533,7 @@ zipp==3.15.0
     #   importlib-resources

 # The following packages are considered to be unsafe in a requirements file:
-pip==23.1.2
-    # via napari-plugin-manager
-setuptools==68.0.0
+setuptools==67.7.2
     # via
     #   nvidia-cublas-cu11
     #   nvidia-cuda-cupti-cu11
--
2.40.0.windows.1
